os/mm/mm_heap/mm_mallinfo.c

```diff
@@ -157,6 +246,7 @@ int mm_mallinfo(FAR struct mm_heap_s *heap, FAR struct mallinfo *info)
 		mm_givesemaphore(heap);
 
 		mvdbg("region=%d node=%p heapend=%p\n", region, node, heap->mm_heapend[region]);
+		DEBUGASSERT(node == heap->mm_heapend[region]);
```

Comment:
As I said before, ASSERT is correct I think. From your reply, you've agreed. If right, could you change it to ASSERT?

Reply:
Done.